test_config.placeholder.json

```diff
@@ -1,9 +1,14 @@
 {
```

Comment:
I mainly just have the same comments on this file as in the PIWebAPI PR. 
- Casing?
- Any reason to use an array?
- `endpoint-type` seems unnecessary?

Reply:
@cbardini I think the only comment I still had on this is that "EndpointType" probably isn't necessary here.

Reply:
@dcbroad3 I removed it from the README, but I forgot to remove it from the actual file. It should be good now.